android/src/main/java/com/gettipsi/stripe/StripeModule.java

```diff
@@ -201,7 +201,7 @@ public void createTokenWithCard(final ReadableMap cardData, final Promise promis
 
       mStripe.createCardToken(
         createCard(cardData),
-        mPublicKey,
+        // mPublicKey,
```

Comment:
Thanks for taking the initiative and putting out a fix for this.

Minor nit, I might recommend removing this parameter altogether (as opposed to commenting it out) and adding a comment referencing #311 for future context.

Reply:
Okay sure, I will  remove the parameter.

Reply:
it's resolved.